src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  height: 406px;
+  box-sizing: border-box;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to group CSS properties in a specific order: positioning, box model, typography, visual, misc. In this case, `box-sizing` should be at the top of the block, as it is a box model property.